shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -66,6 +71,47 @@
    */
   int64_t buttons = 0;
 
+  /**
+   * Pan gesture is currently sending us events.
+   */
+  bool pan_gesture_active = false;
+
+  /**
+   * The accumulated gesture pan.
+   */
+  CGFloat deltaX = 0;
+  CGFloat deltaY = 0;
```

Comment:
I hate to throw on a minor nit at the last moment, but these should be `delta_x` and `delta_y` in raw C/C++ structs (in line with other members here). 

We do use camelCase naming for Objective-C ivars/locals though.